File: app/src/main/java/tpcreative/co/qrscanner/model/DriveAbout.java
package tpcreative.co.qrscanner.model;
import java.io.Serializable;
import java.util.List;
import tpcreative.co.qrscanner.common.api.response.BaseResponseDrive;
import tpcreative.co.qrscanner.common.api.response.DriveResponse;

public class DriveAbout extends BaseResponseDrive implements Serializable{
    public long inAppUsed;
    public DriveUser user;
    public StorageQuota storageQuota;
    /*Create folder*/
    /*Drive api queries*/
    public List<DriveResponse> files;
}

File: app/src/main/java/tpcreative/co/qrscanner/model/DriveAbout.kt
package tpcreative.co.qrscanner.model

import tpcreative.co.qrscanner.common.api.response.BaseResponseDrive
import tpcreative.co.qrscanner.common.api.response.DriveResponse
import java.io.Serializable

class DriveAbout : BaseResponseDrive(), Serializable {
    var inAppUsed: Long = 0
    var user: DriveUser? = null
    var storageQuota: StorageQuota? = null

    /*Create folder*/ /*Drive api queries*/
    var files: MutableList<DriveResponse?>? = null
}